vic/drivers/shared_all/src/calc_root_fraction.c

```diff
@@ -42,6 +42,7 @@ calc_root_fractions(veg_con_struct  *veg_con,
     size_t               layer;
     size_t               zone;
     size_t               i;
+    size_t               max_itter;
```

Comment:
should read `max_iter`. Actually it would make more sense if this were simply called `iter` or `n_iter`, since it is not the maximum number of iterations but just a counter